README.md

```diff
@@ -61,3 +61,13 @@ To release a new version of the updatinate tool, run:
 This will release a stable version of whatever is indicated in the VERSION file. e.g. if VERSION contains `1.0.3-dev`, then version `1.0.3` will be tagged and released, and the VERSION file will be updated to `1.0.4-dev`. To release version `1.1.0` instead, manually edit the VERSION file to `1.1.0-dev` and then run `composer release`.
 
 The updatinate.phar file will be uploaded to GitHub on every release. Rebuild [pantheon-systems/docker-updatinator](https://github.com/pantheon-systems/docker-updatinator) to deploy a new version of the tool to the automation processes.
+
+## Project support levels
+
+| Badge | Description
+| ----- | -----------
+| ![Actively Maintained](https://img.shields.io/badge/pantheon-actively_maintained-yellow)      | Actively maintained
+| ![Minimal Support](https://img.shields.io/badge/pantheon-minimal_support-yellow)      | Minimal support, e.g. examples
+| ![Unsupported](https://img.shields.io/badge/pantheon-unsupported-yellow)      | Unsupported, e.g. end-of-life / no longer relevant
+| ![Unofficial](https://img.shields.io/badge/pantheon-unofficial-yellow)      | Unofficial, e.g. public repo made by employee (hackweek project et. al.)
+
```

Comment:
The white-on-yellow would be an accessibility issue for too low of a contrast ratio (https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=D6AE22). There are additional styles/parameters that we can pass to shields.io to tweak that, I propose:

```suggestion

| Badge | Description
| ----- | -----------
| ![Actively Maintained](https://img.shields.io/badge/pantheon-actively_maintained-yellow)      | Actively maintained
| ![Minimal Support](https://img.shields.io/badge/pantheon-minimal_support-yellow)      | Minimal support, e.g. examples
| ![Unsupported](https://img.shields.io/badge/pantheon-unsupported-yellow)      | Unsupported, e.g. end-of-life / no longer relevant
| ![Unofficial](https://img.shields.io/badge/pantheon-unofficial-yellow)      | Unofficial, e.g. public repo made by employee (hackweek project et. al.)
```

I'm not 100% sold on only one color for the badge colors, but the `for-the-badge` style makes the text much more readable, so it's easier to see/understand at a glance.